pkg/client/msp/ca.go

```diff
@@ -6,6 +6,8 @@ SPDX-License-Identifier: Apache-2.0
 
 package msp
 
+import "github.com/cloudflare/cfssl/csr"
```

Comment:
We don't include third party structures in our high level API. This import shouldn't be here.